grpc/runtime/src/main/scala/io/buoyant/grpc/runtime/H2Headers.scala

```diff
@@ -0,0 +1,21 @@
+package io.buoyant.grpc.runtime
+
+import com.twitter.finagle.buoyant.h2
+
+private[runtime] object H2Headers {
+  private[this] val ContentType = "content-type"
+  private[this] val GrpcProtoContentType = "application/grpc+proto"
+
+  def responseHeaders(http2Status: h2.Status = h2.Status.Ok): h2.Headers = h2.Headers(
+    h2.Headers.Status -> http2Status.toString,
+    ContentType -> GrpcProtoContentType
+  )
+
+  def requestHeaders(path: String): h2.Headers = h2.Headers(
+    h2.Headers.Scheme -> "http",
```

Comment:
maybe we can put that in as val the same way we do for `GrpcProtoContentType`.

Reply:
I decided to change the code slightly. Although `Headers` are mutable, tuples are not.

So I decided declare entire tuples as fields on the object (and maybe, if we are unlucky with JIT, save few heap allocations in runtime). This way, I believe, it's more readable, and gives less reasons to extract string literals.

Reply:
Good thinking!